publisher.rb

```diff
@@ -0,0 +1,59 @@
+require 'aws-sdk-sqs'
+class Publisher
+	# Note: You can store the values of the aws access keys and region list in a config file (json) 
+	# from where you can get the values into the constants.
+	ACCESS_KEY_ID = "AKIAIAGPLQGIY2DYSXHQ" #<your access key id>
+	SECRET_ACCESS_KEY = "4bxwryalQnrY+0FMqQHwDHX+EEyrnWgGTluIsfeQ" #<your secret access key>
+  REGION = "ap-south-1" #<your aws account region>
```

Comment:
@rocky02 - you have just given me access to your credentials.. Delete it right now!